index.html

```diff
@@ -56,6 +56,7 @@ <h1 class="sr-only">Dogecoin</h1>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#about"><span data-i18n="dgc-what-is-doge">What is dogecoin</span></a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#start"><span data-i18n="dgc-get-started-now">How to get started</span></a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="guide.html"><span data-i18n="dgc-guide">Guide</span></a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="#" id="toggleBtn" onclick="toggleDN()">🌙</a></li>
```

Comment:
What about guide.html and support.html?

Reply:
my apologies, will add this tonight and push up changes.

Reply:
```suggestion
                    document.body.style.color = "black";
```
Try that
hope it helps